package.json

```diff
@@ -13,13 +13,15 @@
     "update-approved-identifiers": "ts-node ./scripts/approvedIdentifiersTableParser.ts"
   },
   "dependencies": {
+    "abi-decoder": "github:UMAprotocol/abi-decoder",
```

Comment:
These can rather be installed as `devDependencies`

Try delete them from here, `yarn` and then install again but with `yarn add -D`

Reply:
done.